File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        nums = list(set(nums))
        nums.sort()
        Max = 0
        count = 0
        if(len(nums) == 0):
            return Max
        for i in range(len(nums)):
            if (i ==  len(nums)-1):
                return Max + 1
            if(nums[i] == nums[i+1]-1):
                count +=1
            else:
                count = 0
            Max = max(Max, count)
        return Max + 1

File: Data Structures & Algorithms/longest-repeating-substring-with-replacement/submission-2.java
class Solution {
    public int characterReplacement(String s, int k) {
        int res = 0 ;
        int l = 0 ;
        Map<Character,Integer> count = new HashMap<>();

        for(int r=0; r < s.length() ; r++){
            count.put(s.charAt(r),  count.getOrDefault(s.charAt(r), 0)+1);
            int max = Collections.max(count.values());

            while((r-l+1)- max > k){
                count.put(s.charAt(l), count.get(s.charAt(l))-1);
                l++;
            }
            res = Math.max(r-l+1, res);
        }
        return res;
    }
}